### PR TITLE
DevWs for NIE-TECH CO., LTD. containing containing Z-Wave Metering Switch

### PR DESCRIPTION
**Pull request by sky-nie**

### CODE REVIEW

File: devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy [hunk 1]
```diff
@@ -52,6 +52,7 @@ metadata {
 		fingerprint mfr: "031E", prod: "0002", model: "0001", deviceJoinName: "Inovelli Switch" //US //Inovelli Switch Red Series
 		fingerprint mfr: "0154", prod: "0003", model: "000A", deviceJoinName: "POPP Outlet", ocfDeviceType: "oic.d.smartplug" //EU //POPP Smart Outdoor Plug
 		fingerprint mfr: "010F", prod: "1F01", model: "1000", deviceJoinName: "Fibaro Outlet", ocfDeviceType: "oic.d.smartplug" //EU //Fibaro walli Outlet //Fibaro Outlet
+		fingerprint mfr: "0312", prod: "FF00", model: "FF0E", deviceJoinName: "Minoston Outlet Meter", ocfDeviceType: "oic.d.smartplug" //Mini Smart Plug Meter, MP21ZP
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy]: Please change `deviceJoinName` to `Minoston Outlet`.

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy (hunk 1 above)
Comment (sky-nie) [reply devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy]: OK, I will modify it right away

**Comment by sky-nie** [comment]
@PKacprowiczS 
       I have revised it, and you can review it again at any convenient time.

**Comment by PKacprowiczS** [comment]
@sky-nie it looks like you haven't pushed the change, as I don't see any.

**Comment by sky-nie** [comment]
> @sky-nie it looks like you haven't pushed the change, as I don't see any.

@PKacprowiczS :sorry!I chose the wrong branch to commit.

**Review by PKacprowiczS — APPROVED** [review]
@KKlimczukS

**Review by KKlimczukS — APPROVED** [review]

**Pull request merged**